static/src/javascripts-legacy/projects/commercial/modules/article-body-adverts.js

```diff
@@ -174,7 +174,9 @@ define([
     }
 
     function addSlots(totalCount) {
-        qwery('.ad-slot--inline').forEach(addSlot);
+        qwery('.ad-slot--inline').forEach(function (slot) {
```

Comment:
Nice - so this prevents us calling `adSlot` with the index of the array and thus forcing all but the first ad? Nice catch!